src/app/contexts/ToggleContext/index.tsx

```diff
@@ -31,8 +32,15 @@ const ToggleContextProvider = ({
 }: PropsWithChildren<ToggleContextProviderProps>) => {
   const [toggleState, toggleDispatch] = useReducer(toggleReducer, toggles);
 
+  const toggleContextValue = useMemo(
```

Comment:
We weren't sure of the naming for these values that we have been extracting out to then pass back in. Should we just be calling them `memoizedValue` ?